core/backend/src/Entity.ts

```diff
@@ -86,6 +86,12 @@ export class Entity {
 
   /** @internal */
   public static get protectedOperations(): string[] { return []; }
+
+  /** return whether this Entity class is a subclass of another Entity class
+   * @note the subclass-ness is checked according to JavaScript inheritance, to check the underlying raw EC class's
+   * inheritance, you can use [ECClass.is]($ecschema-metadata)
+   */
+  public static is(otherClass: typeof Entity): boolean { return ClassUtils.isSubclassOf(this, otherClass); }
```

Comment:
This might be more useful if it took a type T and return `this is T`. But also more awkward to call I think, because T would need to be supplied as type argument and function argument.

Reply:
should that generic type argument be trivially inferable by the compiler? I'll try it out and add a type test similar to the ones in ClassUtils.test.ts

Reply:
`this` is not usable because it is a static method according to typescript. Which is probably a missing feature on their part. So I think we should leave it as is.